gyp/pylib/gyp/MSVSSettings_test.py

```diff
@@ -1085,6 +1085,7 @@ def testConvertToMSBuildSettings_full_synthetic(self):
             'GenerateManifest': 'true',
             'IgnoreImportLibrary': 'true',
             'LinkIncremental': 'false'}}
+    self.maxDiff = 9999  # on failure display a long diff
```

Comment:
`self.maxDiff = None` if you don't really want an upper limit?

Reply:
Understood.  These are reasonable limits for these test results.